### DISCUSSION TITLE
Welcome to xnedit Discussions!

### DISCUSSION THREAD

**Discussion by unixwork** [unixwork/xnedit#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yannnnnnnnnnnnnnnn** [comment]
Hello
Is it possible to have  a binary version to download  cause I am not able to compile due to X11 ressources missing
Thanks

**Comment by unixwork** [comment]
Can you try these binaries?
[xnedit-bin-x86_64.tar.gz](https://github.com/unixwork/xnedit/files/10074086/xnedit-bin-x86_64.tar.gz)

**Reply by yannnnnnnnnnnnnnnn** [reply]
yes thanks it works
